tasks/scripts/fly-build

```diff
@@ -27,7 +27,7 @@ pushd concourse
     fi
   fi
 
-  go build --tags netgo -a -ldflags "$ldflags" -o $output/fly ./fly
+  go build --tags 'netgo osusergo' -a -ldflags "$ldflags" -o $output/fly ./fly
```

Comment:
> When cgo is available, cgo-based (libc-backed) code is used by default. This can be overridden by using osusergo build tag, which enforces the pure Go implementation. 

(see https://golang.org/pkg/os/user/)